api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -350,6 +350,38 @@ public void getDateFormat_shouldReturnAPatternWithFourYCharactersInIt() {
 		Assert.assertEquals("tt.MM.uuuu", OpenmrsUtil.getDateFormat(Locale.GERMAN).toLocalizedPattern());
 		Assert.assertEquals("dd-MM-yyyy", OpenmrsUtil.getDateFormat(new Locale("pt", "pt")).toLocalizedPattern());
 	}
+	@Test
+	public void getDateFormat_shouldReturnTrueWhenDateIsNotNull() {
+		Date date = null;
+		try {
+			SimpleDateFormat sdf= new SimpleDateFormat(MM/dd/yyyy);
+			  date=sdf.parse(value);
+			  if(!value.equals(sdf.format(date))) {
+				  date=null;
+			  }
+			  
+		}catch(ParseException ex) {
+			ex.printStackTrace();
+		}
+		if(!date==null) {
+			
+		AssertTrue("isvalid -MM/dd/yyyy with 06/30/2019=true",OpenmrsUtil.getDateFormat(Locale.US).toLocalizedPattern);
+		}else {
+		AssertFalse("isInValid  - yyyy-mm-dd with 2019-06-30",OpenmrsUtil.getDtaeFormat(Locale.US).toLocalizedPattern);
```

Comment:
why place assertion in if statements ??
have you seen this ??
https://wiki.openmrs.org/display/docs/Unit+Testing+Conventions

Reply:
yes i seen through that i will continue to see it more and more